### PR TITLE
Create secretive.rb

### PR DESCRIPTION
**Pull request by chriscn**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

**Comment by chriscn** [comment]
The `ci` is failing for the following reason: 'Error: Calling brew pull is deprecated! Use hub checkout instead.' Which I don't believe is an error with the cask

File: Casks/secretive.rb [hunk 1]
```diff
@@ -0,0 +1,10 @@
+cask 'secretive' do
+  version '1.0.2'
+  sha256 '464ab05baaf14246476a04d05b439776ade06e2937477535a7ea68fa947f507f'
+
+  url 'https://github.com/maxgoedjen/secretive/releases/latest/download/Secretive.zip'
```
Comment (vitorgalvao) [inline Casks/secretive.rb:5]: Use the link on [the releases page](https://github.com/maxgoedjen/secretive/releases) and interpolate the version in the URL.

Also, add an [`appcast`](https://github.com/homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/appcast.md).

**Review by vitorgalvao — CHANGES_REQUESTED** [review]

**Review by vitorgalvao — APPROVED** [review]

**Comment by chriscn** [comment]
@vitorgalvao was just about to commit that change, beat me to it!

**Comment by ran-dall** [comment]
@chriscn Have you checked if we need a `zap` stanza? I don't have `Catalina` installed, so I can't do it for you. You can use @nrlquaker's [`createzap` project](https://github.com/nrlquaker/homebrew-createzap), if you need some help.

**Comment by chriscn** [comment]
Should `zap` be repeated twice?

**Comment by ran-dall** [comment]
> Should zap be repeated twice?

@chriscn No, it was a glitch on my end.

**Pull request merged**